cmd/find-main.go

```diff
@@ -123,37 +123,34 @@ FORMAT
       {url} --> Substitutes to a shareable URL of the path.
 
 EXAMPLES:
-   01. Find all files named foo from all buckets.
-       $ {{.HelpName}} s3 --name "file"
+   01. Find all "foo.jpg" from all buckets from "s3" account
+       $ {{.HelpName}} s3 --name "foo.jpg"
 
-   02. Find all text files from mybucket.
+   02. Find all objects ending with ".txt" extension under "mybucket"
        $ {{.HelpName}} s3/mybucket --name "*.txt"
 
-   03. Print only the object names without the directory component under this bucket.
-       $ {{.HelpName}} s3/bucket --name "*" -print {base}
+   03. Print only the object names without the directory component under "mybucket".
+       $ {{.HelpName}} s3/mybucket --name "*" -print {base}
 
-   04. Copy all jpeg files from AWS S3 "s3/photos" bucket to minio "play/photos" bucket.
-       $ {{.HelpName}} s3/photos --name "*.jpg" --exec "mc cp {} play/photos"
-
-   05. Find all jpeg images from any folder prefixed with "album".
+   04. Find all jpg images from any folder prefixed with "album".
        $ {{.HelpName}} s3/photos --name "*.jpg" --path "*/album*/*"
 
-   06. Find all files with extensions jpeg, png, and gif using regex.
+   05. Find all objects with extensions jpg, png, and gif using regex.
        $ {{.HelpName}} s3/photos --regex "(?i)\.(jpg|png|gif)$"
 
-   07. Find all jpeg images from "s3/bucket" and copy to minio "play/bucket" *continuously*.
+   06. Find all jpg images from "s3/bucket" and copy to minio "play/bucket" *continuously*.
```

Comment:
`Find all jpg images from "s3/bucket" and copy to minio "play/bucket" *continuously*.`
 =. 
`Find all objects with ".jpg" extension under "s3/bucket" and copy to minio "play/bucket" *continuously*.`

Reply:
It kept on purpose to indicate that they are images, we don't need to say just extension.